### PR TITLE
Added group purge command and fixed usage descriptions

### PR DESCRIPTION
**Pull request by ariestis**

### CODE REVIEW

File: commands/feedback.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 module.exports = {
 	help: cfg => "Get a link to the official support server!",
-	usage: cfg => ["feedback - get a link to the support server"],
+	usage: cfg => [usage: "feedback", params: "", description: "Get a link to the support server."],
```
Comment (Keterr) [inline commands/feedback.js:3]:
```suggestion
	usage: cfg => [{usage: "feedback", params: "", description: "Get a link to the support server."}],
```

File: commands/group.js [hunk 2]
```diff
@@ -154,6 +156,36 @@ module.exports = {
 			await bot.db.updateGroup(msg.author.id, group.name, "description", description.slice(0,2000));
 			if(description.length > 2000) return "Description updated, but was cut to 2000 characters to fit within Discord embed limits.";
 			return "Description updated.";
+			
+		case "purge":
+			if(!args[1]) return "No group name given.";
+			if(args[1] == "*") {
+				try {
+				await bot.send(msg.channel, `Warning: This will remove the group along with all of the ${cfg.lang}s within it. Reply 'yes' to continue or anything else to cancel.`);
```
Comment (Keterr) [inline commands/group.js:164]: This warning message should be edited to reflect that `tul!group purge *` will remove absolutely everything - all members and all groups

File: commands/group.js [hunk 3]
```diff
@@ -154,6 +156,36 @@ module.exports = {
...
+				let response = await bot.waitMessage(msg);
+				if(response.content.toLowerCase() != "yes") return "Canceling operation.";
+			} catch(e) {
+				if(e == "timeout") return "Response timed out. Canceling.";
+				else throw e;
+			}
```
Comment (Keterr) [inline commands/group.js:170]: indentation wrong here

File: commands/group.js [hunk 4]
```diff
@@ -154,6 +156,36 @@ module.exports = {
...
+				await bot.db.query("DELETE FROM Members WHERE group_id = $1", [msg.author.id]);
+				await bot.db.query("DELETE FROM Groups WHERE user_id = $1",[msg.author.id]);
+				return "All groups deleted and members set to no group.";
```
Comment (Keterr) [inline commands/group.js:173]:
```suggestion
				return "All groups and members deleted.";
```

File: commands/group.js [hunk 5]
```diff
@@ -154,6 +156,36 @@ module.exports = {
...
+			}
+			name = args.slice(1).join(" ");
+				try {
```
Comment (Keterr) [inline commands/group.js:176]: indentation

File: commands/group.js [hunk 6]
```diff
@@ -154,6 +156,36 @@ module.exports = {
...
+				await bot.send(msg.channel, `Warning: This will remove the group along with all of the ${cfg.lang}s within it. Reply 'yes' to continue or anything else to cancel.`);
+				let response = await bot.waitMessage(msg);
+				if(response.content.toLowerCase() != "yes") return "Canceling operation.";
+			} catch(e) {
+				if(e == "timeout") return "Response timed out. Canceling.";
+				else throw e;
+			}
+				await bot.db.query("DELETE FROM Members WHERE group_id = $1", [msg.author.id]);
+				await bot.db.query("DELETE FROM Groups WHERE user_id = $1",[msg.author.id]);
```
Comment (Keterr) [inline commands/group.js:172]:
```suggestion
				await bot.db.members.clear(msg.author.id);
				await bot.db.groups.deleteAll(msg.author.id);
```

File: commands/group.js [hunk 7]
```diff
@@ -154,6 +156,36 @@ module.exports = {
...
+				let response = await bot.waitMessage(msg);
+				if(response.content.toLowerCase() != "yes") return "Canceling operation.";
+			} catch(e) {
+				if(e == "timeout") return "Response timed out. Canceling.";
+				else throw e;
+			}
+			existing = await bot.db.getGroup(msg.author.id, name);
+			if(!existing) return "You don't have a group with that name.";
+			await bot.db.query("DELETE FROM Members WHERE group_id = $1", [existing.id]);
+			await bot.db.deleteGroup(msg.author.id, name);
```
Comment (Keterr) [inline commands/group.js:187]: move this into a `bot.db.groups.purge(groupID)`

File: commands/privacy.js [hunk 8]
```diff
@@ -4,9 +4,9 @@ fs.readFile("./privacy.txt").then(file => policy = file.toString());
 
 module.exports = {
 	help: cfg => "View my privacy policy.",
-	usage: cfg => ["privacy - show the privacy policy"],
+	usage: cfg => [usage: "privacy", params: "", description: "Show the privacy policy"],
```
Comment (Keterr) [inline commands/privacy.js:7]:
```suggestion
	usage: cfg => [{usage: "privacy", params: "", description: "Show the privacy policy"}],
```

**Review by Keterr — CHANGES_REQUESTED** [review]

File: commands/group.js (hunk 2 above)
Comment (ariestis) [reply commands/group.js:164]: oh shit does it remove ALL of them? I thought I made it so that it only removed the one group and all members in it

Comment (Keterr) [reply commands/group.js:164]: this is within the `tul!group purge *` case which would be a command to "purge all groups", the code for purging a single group is down below

Comment (ariestis) [reply commands/group.js:164]: oh wait duh lol Im dumb and have no reading comprehension

**Pull request closed without merging**